change/@azure-msal-common-e77b173d-7e21-4295-82f2-5dcc3edf653f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Patch is fine for this change
```suggestion
  "type": "patch",
```

---

I think this should be minor, we are adding something to the public surface.